c_reference/tests/utils/test_quantized_utils.c

```diff
@@ -2,12 +2,13 @@
 // Licensed under the MIT license.
```

Comment:
In this file, the tests are hard to understand since the matrices are directly written in in unravelled form. Please use a format similar to the following:

INT_T matrix1[1][4][4][3] = {
     {
           {1,2,3},
           {4,5,6},
           {7,8,9},
           {10,11,12}
     },
     {
           {13,14,15},
           {16,17,18},
           {19,20,21},
           {22,23,24}
     },
     {
           {25,26,27},
           {28,29,30},
           {31,32,33},
           {34,35,36}
     },
     {
           {37,38,39},
           {40,41,42},
           {43,44,45},
           {46,47,48}
     }
};

then when calling the function, you can use:
Function(..., &matrix1[0][0][0][0], ...);
to unravel the matrix to the format required by the function

Reply:
Hi Aayan, I prefer not to have nested brackets in initialization since I am not sure whether all compilers will pick it up. For the same reason, I suggested  using notation mat[1*4*4*3] = {...}. At the same time, retaining the indented formatting for matrix is fine.

Reply:
Ok, then let's have indented matrices like the following

int A[2][2][2][2] = {
    1,2,
    3,4,

    5,6,
    7,8,
//////////////
    9,10,
    11,12,

    13,14,
    15,16
};

(In any way all 4 dimensions can be discernable)